huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_topic.go

```diff
@@ -91,7 +92,7 @@ func ResourceDmsRocketMQTopic() *schema.Resource {
 	}
 }
 
-func DmsRocketMQTopicBrokerRefSchema() *schema.Resource {
+func RocketMQTopicBrokerRefSchema() *schema.Resource {
```

Comment:
```suggestion
func rocketMQTopicBrokerRefSchema() *schema.Resource {
```
it seems shoule be a private func

Reply:
done